src/WowzaApiClient.php

```diff
@@ -18,45 +16,29 @@ class WowzaApiClient
     const OPTION_VERSION = 'version';
     const OPTION_APPEND = 'append';
     const OPTION_OVERWRITE = 'overwrite';
+    const CONNECTION_TIMEOUT = 5;
```

Comment:
in the code I have noticed that it is used as default and it can be rewritten. am I right?. can we add this to the name? DEFAULT_CONNECTION_TIMEOUT

Reply:
Yes sure we can do this